Gaijinizer patch/Map217.txt

```diff
@@ -1688,301 +1688,393 @@ Somehow, it has been lost since then.#GT
+B-But, wasn't she really happy back then?!
 
+How could she end up going on a rampage?!#RonIffy
 > END STRING
 
 > BEGIN STRING
 ……………レイナと同じだよ。
 
 彼女は間一髪のところで、仲間たちの力によって
 魔力の渦から引き上げられた
-> CONTEXT: Map217/events/5/pages/2/708/Dialogue/hyakki < UNTRANSLATED
+> CONTEXT: Map217/events/5/pages/2/708/Dialogue/hyakki
+It was the same as with Reina.
 
+She was pulled out from a magic vortex by her
+friends in the nick of time.#Ron
 > END STRING
 
 > BEGIN STRING
 だけど………その引き換えに多くの犠牲が出た……
 
 直前まで安定していて、魔力の暴走も小規模だった
  アンタほどうまくはいかなかったんだよ
-> CONTEXT: Map217/events/5/pages/2/713/Dialogue/hyakki < UNTRANSLATED
+> CONTEXT: Map217/events/5/pages/2/713/Dialogue/hyakki
+But there were a lot of victims left behind in
+exchange for that.
+
 
+Though she was stable right until the last
+minute, and the scale of her rampage was small,
+things didn't work out as well as they did for
+you.#RonIffy
 > END STRING
 
 > BEGIN STRING
 世界の一部は焦土と化し、
 当然、彼女を助けようとした仲間たちも……
 
 魔力を使い果たし、『個』が消滅してしまった。
-> CONTEXT: Map217/events/5/pages/2/718/Dialogue/hyakki < UNTRANSLATED
-
+> CONTEXT: Map217/events/5/pages/2/718/Dialogue/hyakki
+Part of the world was scorched, and her friends
+used up all their magic trying to save her and
+lost their "selves".#Ron
 > END STRING
 
 > BEGIN STRING
 守っていたはずの神社や街が消滅し、大事な仲間さえ失った…
 
  哀しみに押し潰されそうになった彼女は……
 『ある決断』をしたのサ。
-> CONTEXT: Map217/events/5/pages/2/723/Dialogue/hyakki < UNTRANSLATED
-
+> CONTEXT: Map217/events/5/pages/2/723/Dialogue/hyakki
+The shrine and town she was supposed to protect
+were obliterated, she'd even lost her precious
+friends. And so, crushed by sorrow, she decided
+to do "that".#Ron
 > END STRING
 
 > BEGIN STRING
  『ある決断』……？
 
 いったいどんな……
-> CONTEXT: Map217/events/5/pages/2/728/Dialogue/kazuya < UNTRANSLATED
+> CONTEXT: Map217/events/5/pages/2/728/Dialogue/kazuya
+"That"?
 
+What on earth do you mean?#Ron
 > END STRING
 
 > BEGIN STRING
 『捨てた』んだよ。
 
 これまでの、無邪気で幸せだった自分の『個』を……
 あの何もない、虚数空間の彼方に……
-> CONTEXT: Map217/events/5/pages/2/733/Dialogue/hyakki < UNTRANSLATED
-
+> CONTEXT: Map217/events/5/pages/2/733/Dialogue/hyakki
+She discarded her carefree, happy self and tossed
+it into that space of nothingness.#Ron
 > END STRING
 
 > BEGIN STRING
 彼女は、人間を好きだった感情や、心を捨てて
 
 人間界を悪魔から守る最高責任職『長（おさ）』として
 真剣に自分の世界を守る道を選んだのさ。
-> CONTEXT: Map217/events/5/pages/2/738/Dialogue/hyakki < UNTRANSLATED
-
+> CONTEXT: Map217/events/5/pages/2/738/Dialogue/hyakki
+She discarded her feelings and love for humanity,
+and chose to take up supreme responsibility for
+earnestly defending the human realm, her world,
+from demons.#RonIffy
 > END STRING
 
 > BEGIN STRING
 だから、アンタが出会った『夜深』は、
 『本物』には違いないけど、正確には『本人』じゃない。
 
  虚数空間を漂う、過去の彼女の『個』なんだよ。
-> CONTEXT: Map217/events/5/pages/2/743/Dialogue/hyakki < UNTRANSLATED
-
+> CONTEXT: Map217/events/5/pages/2/743/Dialogue/hyakki
+So while the Yami you met in there is definitely
+the real her, it's not exactly the "true" her,
+but her "self" from the past floating in empty,
+imaginary space.#Ron
 > END STRING
```

Comment:
Some slight rewording for flow:
```
So while the Yami you met in there is definitely
the real deal, it's not exactly her true "self", but
the castoffs from the past floating in empty, 
imaginary space.
```

Reply:
Looks good, but I think it'd be better if we have "castoff" in the singular rather than the plural.

> So while the Yami you met in there is definitely
> the real deal, it's not exactly her true "self". 
> Rather, it's a castoff from the past that's
> been floating in empty, imaginary space.

Reply:
Sounds good to me!